packages/swapper/src/swappers/JupiterSwapper/swapperApi/getTradeRate.ts

```diff
@@ -139,6 +199,40 @@ export const getTradeRate = async (
     {} as Record<AssetId, ProtocolFee>,
   )
 
+  if (feeAccountInstruction) {
+    if (solAsset) {
+      protocolFees[solAssetId] = {
+        requiresBalance: true,
+        amountCryptoBaseUnit: bnOrZero(protocolFees[solAssetId]?.amountCryptoBaseUnit)
+          .plus(PDA_ACCOUNT_CREATION_COST)
+          .toFixed(),
+        asset: solAsset,
+      }
+    }
+  }
+
+  if (input.sendAddress && solAsset) {
```

Comment:
and rm right side if we early return at the beginning of this method `if (!solAsset)`

Reply:
Done in [6840b5c](https://github.com/shapeshift/web/pull/8267/commits/6840b5c3ff9eefd79239117bbd0c17cdb1c4bebf)